src/ompl/tools/thunder/src/SPARSdb.cpp

```diff
@@ -256,17 +269,25 @@ bool ompl::geometric::SPARSdb::getPaths(const std::vector<Vertex> &candidateStar
                                         const base::State *actualGoal, CandidateSolution &candidateSolution,
                                         const base::PlannerTerminationCondition &ptc)
 {
+    std::ofstream nbhd_size;
+    nbhd_size.open("zzz_nbhd_size.txt", std::ios::app);
+    nbhd_size << "\nNeighborhood size is " << candidateStarts.size() << " starts and " << candidateGoals.size() << " goals.\n";
+    int lazyCollisionCheckTime {0};
```

Comment:
what are the units here? put units in the variable name

Reply:
This will be removed